notification-server/Cargo.toml

```diff
@@ -7,5 +7,25 @@ version = "0.1.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-
+anyhow = "1.0.86"
+clap = { version = "4.5.7", features = ["derive", "env"] }
 concordium-rust-sdk = { version = "*", path = "../deps/concordium-rust-sdk" }
+dotenv = "0.15"
+env_logger = "0.10"
+futures = "0.3.30"
+log = "0.4.21"
+tokio = { version = "1.23", features = ["rt-multi-thread", "signal"] }
+tokio-postgres = "0.7"
+tonic = {version = "0.10", features = ["tls", "tls-roots"]} # Use system trust roots.
+tracing-subscriber = { version = "0.3" }
+tracing = { version = "0.1" }
+warp = "0.3"
```

Comment:
We usually use `axum` for http servers (with the reason that it is better maintained I think), it would be nice if we aligned here as well

Reply:
I can tell you are also using warp. The reason is fair enough though